modules/forum/src/main/MentionNotifier.scala

```diff
@@ -0,0 +1,66 @@
+package lila.forum
+
+import lila.common.Future
+
+/**
+  * Notifier to inform users if they have been mentioned in a post
+  *
+  * @param messageApi Api for sending inbox messages
+  */
+final class MentionNotifier(messageApi: lila.message.Api) {
```

Comment:
next step: people complain and require a setting to turn it off.

I don't think PMs are suited for that (or for inviting people in studies). Lichess lacks a general notification system.